gadget/ondisk.go

```diff
@@ -141,7 +139,7 @@ type OnDiskVolume struct {
 	Size Size
 	// sector size in bytes
 	SectorSize     Size
-	partitionTable *sfdiskPartitionTable
+	PartitionTable *sfdiskPartitionTable
```

Comment:
not sure what to do immediately, it relates to the larger TODOs, but having an exported field using an internal type pointer is weird

Reply:
Hmm, yes. Perhaps we could keep it non-exported and provide an UpdatePartitionTable() method in the reorganization.